recipe/meta.yaml

```diff
@@ -23,40 +23,17 @@ requirements:
     - setuptools
     - wheel
   run:
-    - arrow <3.0,>=1.2.0
-    - backoff <4.0,>=2.2.1
-    - beautifulsoup4 <6.0,>=4.8.0
-    - click <10.0
-    - croniter <1.5.0,>=1.3.0
-    - dateutils <2.0
-    - deepdiff <8.0,>=5.7.0
```

Comment:
There's a ModuleNotFoundError at https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=850477&view=logs&j=656edd35-690f-5c53-9ba3-09c10d0bea97&t=986b1512-c876-5f92-0d81-ba851554a0a3.

Reply:
Thanks @weiji14 

Ever since we made the dependencies for the lightning.app submodule optional, we guarded the CLI entry point too. For pip installations, in order to use the CLI entry point, the user has to install the extra as `pip install lightning[app]`. But since the concept of "extras" don't exist for conda installation, we also need to drop the entry point testing here I think.

Reply:
Sure, we can remove the CLI entrypoint, but you should note that if we merge this PR now, there will be no way for anyone to get new versions of the Lightning App CLI when installing from conda-forge, since there is no `lightning-app` package yet (I see @Borda tried adding one at https://github.com/conda-forge/staged-recipes/pull/19716, but closed it for some reason).

Would you like to wait for a `lightning-app` package to be created (containing the dependencies of `pip install lightning[app]`), or merge this directly and handle the `lightning-app` packaging later?

Reply:
Thank you for the help @weiji14, really appreciated! I'll let @Borda comment on that, I'm not 100% sure.